providers/amazon/src/airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -145,7 +145,7 @@ class BatchOperator(AwsBaseOperator[BatchClientHook]):
     def operator_extra_links(self):
         op_extra_links = [BatchJobDetailsLink()]
 
-        if isinstance(self, MappedOperator):
+        if self.is_mapped or isinstance(self, MappedOperator):
```

Comment:
```suggestion
        if self.is_mapped:
```

MappedOperator always has this flag set to True. We can remove the import altogether now too.

Reply:
thank you :) Fixed!